src/main/java/gmailgdogra/ReadXlsx.java

```diff
@@ -18,11 +18,11 @@
     private static final int COL_SWIPE_DATE_TIME = 2;
     private static final int COL_DEVICE_NAME = 5;
 
-    public static List<SwipeRecord> readAllRows(InputStream inputStream) throws IOException {
+    public static List<SwipeRecord> readAllRows(InputStream inputStream) throws Exception {
```

Comment:
Do you use the most general type of exception for purpose? Why?

Reply:
I'll fix it